AzureBasicLoadBalancerUpgrade/module/AzureBasicLoadBalancerUpgrade/modules/ValidateScenario/ValidateScenario.psm1

```diff
@@ -1,6 +1,7 @@
 # Load Modules
 Import-Module ((Split-Path $PSScriptRoot -Parent) + "/Log/Log.psd1")
 Import-Module ((Split-Path $PSScriptRoot -Parent) + "/GetVmssFromBasicLoadBalancer/GetVmssFromBasicLoadBalancer.psd1")
+Import-Module Az.Network
```

Comment:
@cbellee Why do we need to explicitly import the Az.network module here? it is required by the parent module, so must be installed, but we've relied on implicit loading

Reply:
My bad, I added it to prevent an issue where the ALB type wasn't found when the module was loaded, so will remove it.